crates/proof-of-sql/src/base/database/scalar_and_i256_conversions.rs

```diff
@@ -14,7 +14,7 @@ const MAX_SUPPORTED_I256: i256 = i256::from_parts(
 pub fn convert_scalar_to_i256<S: Scalar>(val: &S) -> i256 {
     let is_negative = val > &S::MAX_SIGNED;
     let abs_scalar = if is_negative { -*val } else { *val };
-    let limbs = abs_scalar.into();
+    let limbs: [u64; 4] = abs_scalar.into();
```

Comment:
Is this necessary? No reason to include it in the PR if not.

Reply:
I think so since I added `Into<BigInt>` to `Scalar` definition